SonarQube.TeamBuild.PreProcessor/SonarWebService.cs

```diff
@@ -5,23 +5,20 @@
 // </copyright>
 //-----------------------------------------------------------------------
 
+using Newtonsoft.Json.Linq;
+using SonarQube.Common;
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 using System.Net;
-using Newtonsoft.Json.Linq;
-using SonarQube.Common;
 
 namespace SonarQube.TeamBuild.PreProcessor
 {
-    public sealed class SonarWebService
+    public sealed class SonarWebService : ISonarQubeServer, IDisposable
     {
-        public string Server { get; private set; }
```

Comment:
The server method isn't used publicly. It should probably be pushed inside IDownloader along with the username and password.